src/pages/generate/pages/loading/ProgressBar.tsx

```
@@ -105,7 +105,8 @@ const ProgressLoadingBar = ({ onComplete }: ProgressLoadingBarProps) => {
         style={{
           overflow: 'hidden',
           position: 'relative',
-          height: '2.4rem',
+          minHeight: '2.4rem',
```

ESLint

[error] Replace `'2.4rem'` with `"2.4rem"`

(prettier/prettier)